pkgs/tools/networking/network-manager/l2tp.nix

```diff
@@ -25,6 +25,9 @@ stdenv.mkDerivation rec {
     substituteInPlace ./src/nm-l2tp-service.c \
       --replace /sbin/ipsec  ${strongswan}/bin/ipsec \
       --replace /sbin/xl2tpd ${xl2tpd}/bin/xl2tpd
+
+    substituteInPlace ./autogen.sh \
+      --replace "./configure" ": # ./configure"
```

Comment:
Rather then patching `autogen.sh`, I would add `autoreconfHook` and remove `preConfigure` too.